### PR TITLE
Update fr.yml

### PR DESCRIPTION
**Pull request by trebmuh**
Updated with: 
- the new items you put,
- a few fixes.

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -11,7 +11,14 @@ fr:
     formats:
       default: "%b %d, %Y"
 
+   helpers:
```
Comment (lfzawacki) [inline config/locales/fr.yml]: The problem was the indentation right here, there was one space missing. The translation file uses indentation to create a hierarchy of keys. But yeah, that error message was not helpful.

File: config/locales/fr.yml [hunk 2]
```diff
@@ -274,32 +291,34 @@ fr:
...
-      licensing_rights: "Réflêchissez aux droits que vous donnez aux utilisateurs de la ressource."
+      licensing_rights: "Réfléchissez aux droits que vous donnez aux utilisateurs de la ressource."
       licensing_rights_questions: "Peuvent-ils la commercialiser ? La modifier ? La redistribuer ?"
 
       open_licenses: Lisez à propos des licences ouvertes
       open_licenses_url: 'https://github.com/lfzawacki/musical-artifacts/wiki/Open-Licenses'
 
       open_format: "Format ouvert"
-      open_format_description: "Rendez la disponible sous un format de document commun et ouvertement documenté."
+      open_format_description: "in a common and openly documented format. Rendez-la disponible sous un format documenté ouvertement et commun."
```
Comment (lfzawacki) [inline config/locales/fr.yml]: It seems this english text was extraneous here, I've removed it.

File: config/locales/fr.yml (hunk 1 above)
Comment (trebmuh) [inline config/locales/fr.yml]: OK

File: config/locales/fr.yml (hunk 2 above)
Comment (trebmuh) [inline config/locales/fr.yml]: Correct. Thanks.

**Comment by lfzawacki** [comment]
Latest changes are already on the website. Thanks for the contributions.

**Comment by trebmuh** [comment]
Sweet, thanks for your quickness!

**Pull request merged**